docs/undergrad.yaml

```
@@ -157,7 +157,7 @@ Ring Theory:
   Polynomial rings:
     $K[X]$ is a euclidean ring when $K$ is a field: 'polynomial.euclidean_domain'
     irreducible polynomial: 'irreducible'
-    cyclotomic polynomials in $\Q[X]$: ''
+    cyclotomic polynomials in $\Q[X]$: 'ring_theory/polynomial/cyclotomic'
```

```suggestion
    cyclotomic polynomials in $\Q[X]$: polynomial.cyclotomic'
```
This should be the lean name (including namespaces), not the directory.